python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -279,6 +279,16 @@ def __init__(
         #  by the AMCS.
         self.previous_state = MotionState.PARKED
 
+        # Keep track of the parameters of the latest moveAz command issued so
+        # repetition of the same command can be avoided. This is necessary in
+        # case the dome repeatedly is instructed to move to the same position
+        # with velocity == 0.0 since that may introduce large oscillations. The
+        # position is set to the dome park position.
+        self.latest_moveAz_command = {
+            "position": 360.0 - DOME_AZIMUTH_OFFSET,
```

Comment:
Please consider initializing the position to NaN. That way you will always get "no match" for the first moveAz command, and any reader can instantly see that.

Reply:
Also, as noted in general comments: I suspect you should initialize this when you go out of enabled state, or when disconnected from the low-level controller (which one depends on if you also set it based on telemetry, and in which states you do that).

Reply:
Both done. I opted not to use telemetry since issuing a single moveAz command will not cause vibrations and the hassle of making all the required changes doesn't outweigh handling that case. The `latest_moveAz_command` now is reset when disconnecting from the low-level controller.